libp2p/NodeTable.h

```diff
@@ -195,6 +211,8 @@ class NodeTable : UDPSocketEvents
     static constexpr std::chrono::milliseconds c_evictionCheckInterval{75};
     /// How long to wait for requests (evict, find iterations).
     static constexpr std::chrono::milliseconds c_reqTimeout{300};
+    /// How long to wait before starting a new discovery round
+    static constexpr std::chrono::milliseconds c_discoveryRoundIntervalMs{c_reqTimeout * 2};
```

Comment:
Better define it as `boost::posix_time::milliseconds`, because that's what's actually needed

Reply:
I'm not paying attention to this PR any more, hopefully you will be able to sort it out.
In this case, there is a way to define boost timer with `std::chrono`. You need to use the templeted version `basic_deadline_timer`.

Reply:
If these intervals are used only for boost timers, maybe it's simpler just to use `boost::posix_times`